### DISCUSSION TITLE
Training Corpora Augmentation

### DISCUSSION THREAD

**Discussion by balachander1964** [explosion/spaCy#13115] [Help: Coding & Implementations] [answered]
Hi, I am using Spacy version 3.4 as of now. I am trying to use the multi-step spacy augmentor from create_config file. I request you to provide an example about how to do this. I do not see any guidance or samples or any documentation for this. Also, I request you to let me know the augmentors available already from spacy. Thank you in advance. Bala

**Comment by shadeMe** [comment, marked as answer]
You can take a look at the [the usage documentation](https://spacy.io/usage/training#data-augmentation). Built-in augmenters are documented in the [API section](https://spacy.io/api/top-level#augmenters).

The [Augmenty](https://spacy.io/universe/project/augmenty) library might also be of interest to you.

**Reply by balachander1964** [reply]
Hi, Thank you for providing the link. I request you to let me know where I can get the documentation for augmentee. Thanks once again.
